src/main/resources/static/css/style.css

```
@@ -0,0 +1,216 @@
+:root {
+    --bg-color: #0a0510;
+    --nav-bg: #1a0b2e;
+    --accent-color: #d4af37;
+    --primary-purple: #6a1b9a;
+    --text-color: #f4f4f4;
+}
+
+body {
+    background-color: var(--bg-color);
+    color: var(--text-color);
+    font-family: 'Inter', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)